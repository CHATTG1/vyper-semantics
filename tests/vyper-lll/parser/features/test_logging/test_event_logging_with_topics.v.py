MyLog: __log__({arg1: indexed(bytes <= 3)})

@public
def foo():
    log.MyLog('bar')
