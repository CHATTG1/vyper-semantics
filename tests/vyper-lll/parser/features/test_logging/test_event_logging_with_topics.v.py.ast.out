$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 3264763256 ) , $%seq ( $pass , $assert ( $iszero ( $callvalue ) ) , $%seq ( $pass , $log2 ( 0 , 0 , 59706252898153177344596966318188700657335029269745203515167303800616591592307 , 44498830125527143464827115118378702402016761369235290884359940707316142178304 ) ) , $stop ) ) ) , 0 ) ) )
